sample/src/main/java/kin/sdk/migration/sample/OnBoarding.java

```diff
@@ -20,11 +18,13 @@
 class OnBoarding {
 
     private static final int FUND_KIN_AMOUNT = 10;
-    private static final String TEST_SDK_URL_CREATE_ACCOUNT = "http://friendbot-testnet.kininfrastructure.com?addr=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
-    private static final String TEST_CORE_URL_CREATE_ACCOUNT = "http://friendbot-playground.kininfrastructure.com/?addr=";
-    private static final String TEST_SDK_URL_FUND = "http://friendbot-testnet.kininfrastructure.com/fund?addr=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+    private static final String TEST_SDK_URL_CREATE_ACCOUNT =
+        "https://friendbot-testnet.kininfrastructure.com?addr=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+    private static final String TEST_CORE_URL_CREATE_ACCOUNT = "https://friendbot-playground.kininfrastructure.com/?addr=";
+    private static final String TEST_SDK_URL_FUND =
+        "https://friendbot-testnet.kininfrastructure.com/fund?addr=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
     private static final String TEST_CORE_URL_FUND =
-        "http://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+        "https://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
```

Comment:
same as kin `kin-core`, there's no https for playground faucet

Reply:
Will fix